smithy-rules-engine/src/main/resources/META-INF/smithy/smithy.rules.smithy

```diff
@@ -122,8 +122,9 @@ list OperationInputs {
 @unstable
 @private
 structure OperationInput {
-    /// The name of the service operation targeted by the test.
+    /// The service operation targeted by the test.
     @required
+    @idRef(failWhenMissing: true, selector: "operation")
```

Comment:
does this work when the operationName is not the full shape name? Are we counting on a tool to transform this?

Reply:
it doesn't—maybe it's better to do it manually so we can emit a warning instead

Reply:
Yeah - I think a warning here might be preferable.

Reply:
`idRef` is for absolute shape IDs, so just a name that doesn't resolve to a shape ID according to [the IDL resolution rules](https://smithy.io/2.0/spec/idl.html#relative-shape-id-resolution) won't work.